### DISCUSSION TITLE
Error 54 in index php

### DISCUSSION THREAD

**Discussion by Arhyus** [opensourcepos/opensourcepos#4020] [Q&A]
Hello everyone, I use php 8.1, wamp x64, myadminsql. I’ve set everything up, but when I want to log into localhost I get this error, tell me what’s causing it. Installing on windows 11 x64
![3444](https://github.com/user-attachments/assets/8c0accb1-094a-46b1-bb62-dae7d806cc20)

**Comment by SteveIreland** [comment]
It looks like you're trying to install the main branch which is still in development and you didn't run or didn't successfully complete the build steps.

The main branch isn't ready for production use and has numerous issues that you're going to stumble across.  We are working on them.

**Comment by Arhyus** [comment]
Hi, OK I know you. Then what a version need use for stability work?
--
Отправлено из Mail.ru для Android суббота, 13 июля 2024г., 17:37 +05:00 от Steve Ireland  ***@***.*** :

**Comment by SteveIreland** [comment]
The latest release is found at: https://github.com/opensourcepos/opensourcepos/releases

Installation instructions are found at: https://github.com/opensourcepos/opensourcepos/blob/master/INSTALL.md

**Reply by Arhyus** [reply]
I do everything according to the installation instructions on Windows. I downloaded the latest version of wamp, downloaded the latest release, changed the database.db file in app/config , but when I start the server I can’t now get into localhost/phpmyadmin. Although the login and password that I made in the database.db file I use is the same

**Reply by Arhyus** [reply]
Do you happen to have an assembly configured that I can place in the www folder in the wamp application and launch it?

**Reply by Arhyus** [reply]
and if i want login in phpmyadmin i look this error. 
![image](https://github.com/user-attachments/assets/2f130d1b-8428-4688-9dd3-ee17b1c794f5)

**Reply by A7medAmine** [reply]
I will suggest using wamp server to install ospos just use the 7.4.33 php version wich is provided at the php version list and create the database and you good to go , you dont need to configure the php extension cz its already configured .

**Reply by Arhyus** [reply]
in wamp i use 7.4.33 , and problem saved

**Reply by Arhyus** [reply]
State of services:

 The service 'wampapache64' is started
 Start type : DEMAND_START
 Service Session : LocalSystem

 The service 'wampmysqld64' is NOT started
 EXIT error code:1067
 Help message for error code 1067 is:The process terminated unexpectedly.

 The service 'wampmariadb64' is not RUNNING nor STOPPED.
 [SC] EnumQueryServicesStatus:OpenService failure(s) 1060 :
 The specified service does not exist as an installed service.
 ********* The service 'wampmariadb64' does not exist ********

WampServer (Apache, PHP and MySQL) will not function properly if any service
'wampapache64'
'wampmysqld64'
'wampmariadb64'
 is not started.

        all services BINARY_PATH_NAME are OK
        all services START_TYPE are OK

--- Do you want to copy the results into Clipboard?
--- Press the Y key to confirm - Press ENTER to continue...

Problem in wamp

**Reply by A7medAmine** [reply]
Try to reinstall it in nother disk or check if there is another sql service is on u can check it on the services

**Comment by Arhyus** [comment]
Today i reload with x64 version wamp on x32 wamp, and all services is starting, problem with mysql saved.
--
Отправлено из Mail.ru для Android понедельник, 15 июля 2024г., 20:41 +05:00 от AHMEDabdamine  ***@***.*** :

**Reply by A7medAmine** [reply]
Great ! Is ospos working know !

**Comment by Arhyus** [comment]
Ospos not work, problem on step where need come in mysql with wamp, and redaction sql database.
Phpmyadmin say what HY000, how I last chance.
--
Отправлено из Mail.ru для Android понедельник, 15 июля 2024г., 20:46 +05:00 от AHMEDabdamine  ***@***.*** :

**Reply by A7medAmine** [reply]
If you have discord lets go there and stream your pc for me i would help

**Reply by Arhyus** [reply]
> Если у тебя есть Discord, зайди туда и транслируй свой ПК для меня, я помогу.

Brother, i'm ready my nickname in discord : arhyus , oldname arhyus

**Reply by A7medAmine** [reply]
I apologize to you mate i cannot joing you know i have some jobs to do ! But i promise when ever iam free i will call you just be patient

**Comment by Arhyus** [comment]
Ok, please stay on connect after 1.5 hour's
--
Отправлено из Mail.ru для Android понедельник, 15 июля 2024г., 21:27 +05:00 от AHMEDabdamine  ***@***.*** :

**Comment by Arhyus** [comment]
Ok thank you,  what is your UTC +- ?
--
Отправлено из Mail.ru для Android вторник, 16 июля 2024г., 04:26 +05:00 от AHMEDabdamine  ***@***.*** :

**Reply by A7medAmine** [reply]
I am from Algeria so iam gmt +1

**Reply by A7medAmine** [reply]
Check discord
